cognee/tasks/translation/translate_content.py

```
@@ -0,0 +1,806 @@
+            key = p.strip().lower()
+            if key in _provider_registry and key not in seen:
+                fallback_providers.append(key)
+                seen.add(key)
+            else:
+                invalid_providers.append(p)
+    if invalid_providers:
+        logger.warning("Ignoring unknown fallback providers: %s", invalid_providers)
+    return primary_key, fallback_providers
+
+
+async def _process_chunk(chunk, target_language, primary_key, fallback_providers, confidence_threshold, provider_cache):
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)